metamodel_generator/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/openshift-online/ocm-api-model/metamodel_generator
 
 go 1.23.0
 
-require github.com/openshift-online/ocm-api-metamodel v0.0.69-0.20250609135152-2defa0ded8b3
+require github.com/openshift-online/ocm-api-metamodel v0.0.69-0.20250612120255-488a7c2d8750
```

Comment:
This seems a change in behavior.

Before we were creating new releases in the ocm-api-metamodel but now I observe that we are referencing the same tag but with a different commit.

Before we didn't need to specify a timestamp. Rather we created new tags that could be used as a dependency.

Is this intentional?

Reply:
Im not sure why this is the case now, I noticed this in the last update to the metamodel. @ahitacat mentioned we no longer need to do releases for the metamodel, and i'm unsure why. 

Do you know why this is the case @ahitacat, @nimrodshn?

Reply:
It is not required to do a release. 
But after talking this with @miguelsorianod we think we should keep having the same workflow, and create a new release on the `metamodel`.

Reply:
This has been updated to use v0.0.69